doc/release-notes/5.9-release-notes.md

```diff
@@ -0,0 +1,157 @@
+
+## New JVM Options and DB Settings
+
+The following DB settings allow configuration of the external metadata validator:
+
+- :DataverseMetadataValidatorScript
+- :DataverseMetadataPublishValidationFailureMsg
+- :DataverseMetadataUpdateValidationFailureMsg
+- :DatasetMetadataValidatorScript
+- :DatasetMetadataValidationFailureMsg
+- :ExternalValidationAdminOverride
+
+See the [Database Settings](https://guides.dataverse.org/en/5.9/installation/config.html) section of the Guides for more information.
+
+## Notes for Developers and Integrators
+
+Two sections of the Developer Guide have been updated:
+
+- Instructions on how to sync a PR in progress with develop have been added in the version control section
+- Guidance on avoiding ineffeciencies in JSF render logic has been added to the "Tips" section
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.9 Milestone](https://github.com/IQSS/dataverse/milestone/100?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Community Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](https://guides.dataverse.org/en/5.9/installation/). Please also contact us to get added to the [Dataverse Project Map](https://guides.dataverse.org/en/5.9/installation/config.html#putting-your-dataverse-installation-on-the-map-at-dataverse-org) if you have not done so already.
+
+## Upgrade Instructions
+
+0\. These instructions assume that you've already successfully upgraded from Dataverse Software 4.x to Dataverse Software 5 following the instructions in the [Dataverse Software 5 Release Notes](https://github.com/IQSS/dataverse/releases/tag/v5.0). After upgrading from the 4.x series to 5.0, you should progress through the other 5.x releases before attempting the upgrade to 5.9.
+
+If you are running Payara as a non-root user (and you should be!), **remember not to execute the commands below as root**. Use `sudo` to change to that user first. For example, `sudo -i -u dataverse` if `dataverse` is your dedicated application user.  
+
+In the following commands we assume that Payara 5 is installed in `/usr/local/payara5`. If not, adjust as needed.
+
+`export PAYARA=/usr/local/payara5`
+
+(or `setenv PAYARA /usr/local/payara5` if you are using a `csh`-like shell)
+
+1\. Undeploy the previous version.
+
+- `$PAYARA/bin/asadmin list-applications`
+- `$PAYARA/bin/asadmin undeploy dataverse<-version>`
+
+2\. Stop Payara and remove the generated directory
+
+- `service payara stop`
+- `rm -rf $PAYARA/glassfish/domains/domain1/generated`
+
+3\. Start Payara
+
+- `service payara start`
+  
+4\. Deploy this version.
+
+- `$PAYARA/bin/asadmin deploy dataverse-5.9.war`
+
+5\. Restart payara
+
+- `service payara stop`
+- `service payara start`
+
+6\. Kick off full reindex
```

Comment:
Dumb Don question: why are we re-indexing?

Reply:
@donsizemore - this was the note that I had:

> A full re-index is needed to update the facets, as well as optionally running the "redetect file type" API on existing GeoJSON files.

Reply:
@donsizemore I'll close this out for now but let me know if there are other questions

Reply:
Ok, so it was from the GeoJSON mime type PR - but there’s zero reason to force it on everybody.
It sounds like it should be part of the "Additional Release Steps". "Step 1. - if you have any GeoJSON files misidentified as "type unknown", re-detect them following ... *and then run full re-index, to update the type facets, etc.*"